docs/Using-cmd-line.md

```diff
@@ -35,21 +31,23 @@ $ mvn install -DskipTests=true
 In addition to publishing the war to the local maven repository, the built war file is copied into the apps directory of the server configuration located in the swagger-sample-wlpcfg directory:
 
 ```text
-swagger-sample-wlpcfg
+target
  +- servers
-     +- swaggerSample                            <-- specific server configuration
-        +- server.xml                          <-- server configuration
-        +- apps                                <- directory for applications
-           +- swagger-sample-application.war      <- sample application
-        +- logs                                <- created by running the server locally
-        +- workarea                            <- created by running the server locally
+    +- liberty
+        +- wlp
+            +- usr
+                +- server
+                    +- swaggerSample                            <-- specific server configuration
+                    +- server.xml                          <-- server configuration
+                    +- apps                                <- directory for applications
+                        +- swagger-sample-application.war      <- sample application
+                    +- logs                                <- created by running the server locally
+                    +- workarea                            <- created by running the server locally
```

Comment:
After this file structure example, I would also like to say..... (fill in the proper server package name).

The packaged server `sample.swagger-0.0.1.zip????` file is created in the target folder that includes the server, application, and configuration.